client/sites-dashboard/components/test/sites-site-plan.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * @jest-environment jsdom
+ */
+import { render } from '@testing-library/react';
+import { SitePlan } from '../sites-site-plan';
+
+const siteOwnerId = 1;
+const otherAdminId = 2;
+
+const stagingSite = {
+	ID: 1,
+	site_owner: siteOwnerId,
+	title: 'Staging Site',
+	is_wpcom_staging_site: true,
+};
+
+const activeBusinessSite = {
+	ID: 1,
+	site_owner: siteOwnerId,
+	title: 'Expired Business Site',
+	plan: {
+		expired: false,
+		product_name_short: 'Business',
+	},
+};
+
+const expiredBusinessSite = {
+	ID: 1,
+	site_owner: siteOwnerId,
+	title: 'Expired Business Site',
+	plan: {
+		expired: true,
+		product_name_short: 'Business',
+	},
+};
+
+describe( '<SitePlan>', () => {
+	test( 'shows "Staging" as label for a staging site', () => {
+		const { container } = render( <SitePlan site={ stagingSite } userId={ siteOwnerId } /> );
+		expect( container.textContent ).toBe( 'Staging' );
+	} );
+
+	test( 'shows "Business" as label for a site with an active Business plan', () => {
+		const { container } = render(
+			<SitePlan site={ activeBusinessSite } userId={ otherAdminId } />
+		);
+		expect( container.textContent ).toBe( 'Business' );
+	} );
+
+	test( 'shows "Business" as label for a site with an expired Business plan', () => {
+		const { container } = render(
+			<SitePlan site={ expiredBusinessSite } userId={ otherAdminId } />
+		);
+		expect( container.textContent ).toBe( 'Business - Expired' );
```

Comment:
Here checking `.textContent` directly works because there's nothing else within the component, but with `@testing-library` you usually search for the element using text/label/aria role/etc. so that the test is more resilient.

For example you can't test the case where the plan has expired and the current user is the site owner. But you could with `getByRole` and `getByText`

```ts
		const { getByText, getByRole } = render(
			<SitePlan site={ expiredBusinessSite } userId={ siteOwnerId } />
		);
		expect( getByText( 'Business - Expired' ) ).toBeInTheDocument();
		expect( getByRole( 'link', { name: 'Renew plan' } ) ).toHaveAttribute(
			'href',
			expect.stringMatching( /\/checkout\// )
		);
```